### ISSUE TITLE
The modified image will return to its original state when sent!

### ISSUE THREAD

**Issue by ortwinbachmann30** [forkgram/tdesktop#91]
### Steps to reproduce

I don't know!

### Expected behaviour

Send modified photo!

### Actual behaviour

The modified image will return to its original state when sent!

### Operating system

Windows 11

### Version of Telegram Desktop

4.5.7 beta x64

### Installation source

Static binary from official website

### Logs

```shell
[2023.01.28 15:00:22] Map read time: 167
[2023.01.28 15:00:22] App Info: reading encrypted mtp config...
[2023.01.28 15:00:23] Export Info: Destroy top bar by controller removal.
[2023.01.28 15:00:24] OpenGL: Using custom 'gpu_driver_bug_list.json'.
[2023.01.28 15:00:25] OpenGL Profile: OpenGLES.
[2023.01.28 15:00:25] OpenGL Renderer: ANGLE (Intel, Intel(R) UHD Graphics 610 Direct3D11 vs_5_0 ps_5_0, D3D11-27.20.100.9316)
[2023.01.28 15:00:25] OpenGL Vendor: Google Inc. (Intel)
[2023.01.28 15:00:25] OpenGL Version: OpenGL ES 3.0.0 (ANGLE 2.1.7 git hash: 0bb011f9e403)
[2023.01.28 15:00:25] OpenGL Extensions: GL_OES_EGL_image, GL_OES_compressed_ETC2_sRGB8_alpha8_texture, GL_EXT_texture_norm16, GL_EXT_blend_minmax, GL_NV_pixel_buffer_object, GL_OES_depth24, GL_EXT_blend_func_extended, GL_EXT_texture_storage, GL_EXT_unpack_subimage, GL_OES_compressed_ETC2_sRGB8_texture, GL_ANGLE_program_cache_control, GL_EXT_EGL_image_external_wrap_modes, GL_EXT_sRGB, GL_ANGLE_client_arrays, GL_ANGLE_texture_compression_dxt3, GL_EXT_frag_depth, GL_CHROMIUM_sync_query, GL_OES_texture_float_linear, GL_ANGLE_memory_size, GL_EXT_read_format_bgra, GL_NV_pack_subimage, GL_OES_texture_stencil8, GL_CHROMIUM_bind_generates_resource, GL_OES_element_index_uint, GL_ANGLE_depth_texture, GL_OES_EGL_image_external_essl3, GL_EXT_draw_buffers, GL_ANGLE_pack_reverse_row_order, GL_ANGLE_multiview_multisample, GL_EXT_occlusion_query_boolean, GL_EXT_texture_compression_dxt1, GL_OES_texture_float, GL_ANGLE_lossy_etc_decode, GL_CHROMIUM_copy_compressed_texture, GL_OVR_multiview2, GL_CHROMIUM_bind_uniform_location, GL_EXT_texture_filter_anisotropic, GL_ANGLE_base_vertex_base_instance, GL_OES_compressed_EAC_R11_signed_texture, GL_EXT_shader_texture_lod, GL_ANGLE_framebuffer_blit, GL_OES_EGL_image_external, GL_OES_compressed_ETC2_RGB8_texture, GL_OES_rgb8_rgba8, GL_CHROMIUM_copy_texture, GL_OES_get_program_binary, GL_OES_packed_depth_stencil, GL_ANGLE_texture_compression_dxt5, GL_EXT_robustness, GL_EXT_texture_compression_rgtc, GL_EXT_debug_label, GL_ANGLE_framebuffer_multisample, GL_OES_standard_derivatives, GL_EXT_multisampled_render_to_texture, GL_EXT_texture_compression_s3tc_srgb, GL_ANGLE_explicit_context_gles1, GL_ANGLE_request_extension, GL_NV_fence, GL_OES_compressed_EAC_R11_unsigned_texture, GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture, GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture, GL_OES_fbo_render_mipmap, GL_ANGLE_copy_texture_3d, GL_ANGLE_translated_shader_source, GL_OES_draw_buffers_indexed, GL_EXT_map_buffer_range, GL_OES_depth32, GL_EXT_draw_elements_base_vertex, GL_WEBGL_video_texture, , GL_EXT_color_buffer_float, GL_KHR_debug, GL_NV_EGL_stream_consumer_external, GL_OES_texture_npot, GL_ANGLE_robust_client_memory, GL_CHROMIUM_lose_context, GL_EXT_float_blend, GL_OVR_multiview, GL_OES_texture_border_clamp, GL_ANGLE_instanced_arrays, GL_ANGLE_get_tex_level_parameter, GL_ANGLE_get_serialized_context_string, GL_KHR_parallel_shader_compile, GL_OES_compressed_EAC_RG11_signed_texture, GL_OES_surfaceless_context, GL_OES_texture_half_float_linear, GL_ANGLE_texture_usage, GL_OES_compressed_EAC_RG11_unsigned_texture, GL_OES_vertex_array_object, GL_EXT_draw_buffers_indexed, GL_EXT_texture_rg, GL_EXT_texture_format_BGRA8888, GL_EXT_texture_compression_bptc, GL_ANGLE_texture_multisample, GL_EXT_discard_framebuffer, GL_EXT_debug_marker, GL_ANGLE_explicit_context, GL_ANGLE_provoking_vertex, GL_EXT_color_buffer_half_float, GL_EXT_instanced_arrays, GL_OES_compressed_ETC2_RGBA8_texture, GL_ANGLE_multi_draw, GL_OES_mapbuffer, GL_EXT_disjoint_timer_query, GL_OES_texture_half_float, GL_OES_draw_elements_base_vertex
[2023.01.28 15:00:25] EGL Extensions: EGL_EXT_create_context_robustness, EGL_ANGLE_d3d_share_handle_client_buffer, EGL_ANGLE_d3d_texture_client_buffer, EGL_ANGLE_surface_d3d_texture_2d_share_handle, EGL_ANGLE_query_surface_pointer, EGL_ANGLE_window_fixed_size, EGL_ANGLE_keyed_mutex, EGL_ANGLE_surface_orientation, EGL_ANGLE_direct_composition, EGL_ANGLE_windows_ui_composition, EGL_NV_post_sub_buffer, EGL_KHR_create_context, EGL_KHR_image, EGL_KHR_image_base, EGL_KHR_gl_texture_2D_image, EGL_KHR_gl_texture_cubemap_image, EGL_KHR_gl_renderbuffer_image, EGL_KHR_get_all_proc_addresses, EGL_KHR_stream, EGL_KHR_stream_consumer_gltexture, EGL_NV_stream_consumer_gltexture_yuv, EGL_ANGLE_flexible_surface_compatibility, EGL_ANGLE_stream_producer_d3d_texture, EGL_ANGLE_create_context_webgl_compatibility, EGL_CHROMIUM_create_context_bind_generates_resource, EGL_CHROMIUM_sync_control, EGL_EXT_pixel_format_float, EGL_KHR_surfaceless_context, EGL_ANGLE_display_texture_share_group, EGL_ANGLE_display_semaphore_share_group, EGL_ANGLE_create_context_client_arrays, EGL_ANGLE_program_cache_control, EGL_ANGLE_robust_resource_initialization, EGL_ANGLE_create_context_extensions_enabled, EGL_ANDROID_blob_cache, EGL_ANDROID_recordable, EGL_ANGLE_image_d3d11_texture, EGL_ANGLE_create_context_backwards_compatible, EGL_KHR_create_context_no_error, EGL_KHR_reusable_sync, 
[2023.01.28 15:00:25] OpenGL: QOpenGLContext created, version: 3.3.
[2023.01.28 15:00:25] OpenGL: [TRUE] (OverlayWidget)
[2023.01.28 15:00:29] Update Error: could not get current version 3
[2023.01.28 15:00:31] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:32] Message Info: bad message notification received (error_code 33) for msg_id = 7193664049401082964, seq_no = 58
[2023.01.28 15:00:32] Message Info: bad message notification received, msgId 7193664049401082964, error_code 33
[2023.01.28 15:00:32] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:32] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:32] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:33] Message Info: bad message notification received (error_code 33) for msg_id = 7193664046402517808, seq_no = 40
[2023.01.28 15:00:33] Message Info: bad message notification received, msgId 7193664046402517808, error_code 33
[2023.01.28 15:00:33] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:35] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:37] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:37] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:37] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:37] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:39] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:39] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:39] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:39] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:41] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:41] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:43] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:44] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:44] Update Error: could not get current version 3
[2023.01.28 15:00:44] RPC Error: request 459 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:45] RPC Error: request 473 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:45] API Warning: not loaded minimal channel applied.
[2023.01.28 15:00:47] Message Info: bad message notification received (error_code 33) for msg_id = 7193664118165699148, seq_no = 10
[2023.01.28 15:00:47] Message Info: bad message notification received, msgId 7193664118165699148, error_code 33
[2023.01.28 15:00:47] Message Info: bad message notification received (error_code 33) for msg_id = 7193664118178304704, seq_no = 40
[2023.01.28 15:00:47] Message Info: bad message notification received, msgId 7193664118178304704, error_code 33
[2023.01.28 15:00:47] Message Info: bad message notification received (error_code 33) for msg_id = 7193664118253467640, seq_no = 728
[2023.01.28 15:00:47] RPC Error: request 510 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:47] RPC Error: request 453 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:47] Message Info: bad message notification received (error_code 33) for msg_id = 7193664118721919916, seq_no = 730
[2023.01.28 15:00:47] Message Info: bad message notification received, msgId 7193664118721919916, error_code 33
[2023.01.28 15:00:48] Message Info: bad message notification received (error_code 33) for msg_id = 7193664122545423084, seq_no = 14
[2023.01.28 15:00:48] Message Info: bad message notification received, msgId 7193664122545423084, error_code 33
[2023.01.28 15:00:48] MTP Info: -404 error received in 160004 with temporary key, assuming it was destroyed.
[2023.01.28 15:00:48] RPC Error: request 497 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:49] RPC Error: request 530 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:49] RPC Error: request 526 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:49] RPC Error: request 558 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:50] RPC Error: request 575 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:53] RPC Error: request 609 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:57] RPC Error: request 625 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:00:57] RPC Error: request 628 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:01:06] RPC Error: request 662 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:01:08] RPC Error: request 682 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:01:13] RPC Error: request 718 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:01:30] Skipping message, because it is already in blocks!
[2023.01.28 15:01:46] Message Info: bad message notification received (error_code 33) for msg_id = 7193664372469740864, seq_no = 800
[2023.01.28 15:01:46] Message Info: bad message notification received, msgId 7193664372469740864, error_code 33
[2023.01.28 15:02:04] Skipping message, because it is already in blocks!
[2023.01.28 15:02:11] RPC Error: request 957 got fail with code 400, error USER_NOT_PARTICIPANT
[2023.01.28 15:02:19] RPC Error: request 988 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:02:19] RPC Error: request 989 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:02:20] RPC Error: request 990 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:03:18] Skipping message, because it is already in blocks!
[2023.01.28 15:03:23] RPC Error: request 1431 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:03:31] Message Info: bad message notification received (error_code 33) for msg_id = 7193664824166531320, seq_no = 1332
[2023.01.28 15:03:31] Message Info: bad message notification received, msgId 7193664824166531320, error_code 33
[2023.01.28 15:04:33] Message Info: bad message notification received (error_code 33) for msg_id = 7193665088928723736, seq_no = 266
[2023.01.28 15:04:33] Message Info: bad message notification received, msgId 7193665088928723736, error_code 33
[2023.01.28 15:05:15] API Warning: not loaded minimal channel applied.
[2023.01.28 15:06:30] Message Info: bad message notification received (error_code 33) for msg_id = 7193665591359199572, seq_no = 350
[2023.01.28 15:06:30] Message Info: bad message notification received, msgId 7193665591359199572, error_code 33
[2023.01.28 15:06:41] Message Info: bad message notification received (error_code 33) for msg_id = 7193665638601753524, seq_no = 84
[2023.01.28 15:06:41] Message Info: bad message notification received, msgId 7193665638601753524, error_code 33
[2023.01.28 15:07:41] API Warning: not loaded minimal channel applied.
[2023.01.28 15:07:50] Skipping message, because it is already in blocks!
[2023.01.28 15:08:36] Message Info: bad message notification received (error_code 33) for msg_id = 7193666132509849292, seq_no = 290
[2023.01.28 15:08:36] Message Info: bad message notification received (error_code 33) for msg_id = 7193666132570780216, seq_no = 292
[2023.01.28 15:08:36] Message Info: bad message notification received, msgId 7193666132570780216, error_code 33
[2023.01.28 15:08:42] RPC Error: request 2114 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:08:43] RPC Error: request 2115 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:08:43] RPC Error: request 2117 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:08:43] RPC Error: request 2116 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:09:11] API Warning: not loaded minimal channel applied.
[2023.01.28 15:09:53] Message Info: bad message notification received (error_code 33) for msg_id = 7193666463550379476, seq_no = 202
[2023.01.28 15:09:53] Message Info: bad message notification received, msgId 7193666463550379476, error_code 33
[2023.01.28 15:11:02] RPC Error: request 2517 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:11:55] API Warning: not loaded minimal channel applied.
[2023.01.28 15:12:49] RPC Error: request 2696 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:12:49] RPC Error: request 2695 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 15:13:29] Skipping message, because it is already in blocks!
[2023.01.28 15:13:35] Message Info: bad message notification received (error_code 33) for msg_id = 7193667417217117692, seq_no = 6
[2023.01.28 15:13:35] Message Info: bad message notification received, msgId 7193667417217117692, error_code 33
[2023.01.28 15:14:30] Message Info: bad message notification received (error_code 33) for msg_id = 7193667653655264728, seq_no = 640
[2023.01.28 15:14:30] Message Info: bad message notification received, msgId 7193667653655264728, error_code 33
[2023.01.28 15:14:46] Audio Info: recreating audio device and reattaching the tracks
[2023.01.28 15:14:55] API Warning: not loaded minimal channel applied.
[2023.01.28 15:14:55] API Warning: not loaded minimal channel applied.
[2023.01.28 15:15:58] Skipping message, because it is already in blocks!
[2023.01.28 15:16:01] Audio Info: Closing audio playback device.
[2023.01.28 15:16:02] Audio Info: recreating audio device and reattaching the tracks
[2023.01.28 15:16:29] Audio Info: Closing audio playback device.
[2023.01.28 16:36:42] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:44] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:46] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:47] Message Info: bad message notification received (error_code 33) for msg_id = 7193688836633707372, seq_no = 394
[2023.01.28 16:36:47] Message Info: bad message notification received, msgId 7193688836633707372, error_code 33
[2023.01.28 16:36:47] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:50] API Warning: not loaded minimal channel applied.
[2023.01.28 16:36:51] Message Info: bad message notification received (error_code 33) for msg_id = 7193688874361337028, seq_no = 348
[2023.01.28 16:36:51] Message Info: bad message notification received, msgId 7193688874361337028, error_code 33
[2023.01.28 16:36:51] Message Info: bad message notification received (error_code 33) for msg_id = 7193688875337914028, seq_no = 26
[2023.01.28 16:36:51] Message Info: bad message notification received, msgId 7193688875337914028, error_code 33
[2023.01.28 16:36:52] Message Info: bad message notification received (error_code 33) for msg_id = 7193688879278930896, seq_no = 6
[2023.01.28 16:36:52] Message Info: bad message notification received, msgId 7193688879278930896, error_code 33
[2023.01.28 16:36:52] Message Info: bad message notification received (error_code 33) for msg_id = 7193688879120059016, seq_no = 42
[2023.01.28 16:36:52] Message Info: bad message notification received, msgId 7193688879120059016, error_code 33
[2023.01.28 16:37:28] Could not send ping for some seconds, restarting...
[2023.01.28 16:37:41] App Info: writing encrypted user settings...
[2023.01.28 16:37:51] Message Info: bad message notification received (error_code 33) for msg_id = 7193689127761197188, seq_no = 358
[2023.01.28 16:37:51] Message Info: bad message notification received, msgId 7193689127761197188, error_code 33
[2023.01.28 16:38:06] App Info: writing encrypted user settings...
[2023.01.28 16:38:37] Skipping message, because it is already in blocks!
[2023.01.28 16:38:53] RPC Error: request 3605 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2023.01.28 16:39:44] MTP Info: -404 error received in 160002 with temporary key, assuming it was destroyed.
[2023.01.28 16:39:45] MTP Info: -404 error received in 160004 with temporary key, assuming it was destroyed.
[2023.01.28 16:40:05] API Warning: not loaded minimal channel applied.
[2023.01.28 16:40:05] API Warning: not loaded minimal channel applied.
[2023.01.28 16:40:05] API Warning: not loaded minimal channel applied.
[2023.01.28 16:40:05] API Warning: not loaded minimal channel applied.
[2023.01.28 16:40:05] API Warning: not loaded minimal channel applied.
[2023.01.28 16:40:05] API Warning: not loaded minimal channel applied.
[2023.01.28 16:40:05] API Warning: not loaded minimal channel applied.
[2023.01.28 16:40:05] API Warning: not loaded minimal channel applied.
[2023.01.28 16:42:01] MTP Info: -404 error received in 160001 with temporary key, assuming it was destroyed.
[2023.01.28 16:42:54] App Info: writing encrypted user settings...
[2023.01.28 16:43:10] Skipping message, because it is already in blocks!
[2023.01.28 16:43:33] API Warning: not loaded minimal channel applied.
[2023.01.28 16:44:03] API Warning: not loaded minimal channel applied.
[2023.01.28 16:44:36] API Warning: not loaded minimal channel applied.
[2023.01.28 16:45:27] MTP Info: -404 error received in 160005 with temporary key, assuming it was destroyed.
[2023.01.28 16:45:56] RPC Error: request 4101 got fail with code 400, error PEER_ID_INVALID
[2023.01.28 16:46:50] Message Info: bad message notification received (error_code 33) for msg_id = 7193691447109091504, seq_no = 1290
[2023.01.28 16:46:50] Message Info: bad message notification received, msgId 7193691447109091504, error_code 33
[2023.01.28 16:48:14] API Warning: not loaded minimal channel applied.
[2023.01.28 16:49:14] App Info: writing encrypted user settings...
[2023.01.28 16:49:26] Skipping message, because it is already in blocks!
[2023.01.28 16:52:40] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:40] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:41] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:41] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:41] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:41] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:41] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:41] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:42] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:42] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:42] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:42] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:42] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:42] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:42] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:42] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:43] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:43] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:43] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:43] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:44] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:44] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:44] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:44] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:48] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:48] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:49] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:49] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:49] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:49] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:49] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:49] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:50] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:50] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:50] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:50] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:50] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:50] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:50] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:50] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:51] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:51] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:51] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:51] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:52] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:52] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:52] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:52] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:56] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:56] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:56] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:56] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:57] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:57] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:57] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:57] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:58] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:58] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:58] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:58] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:59] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:59] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:59] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:59] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:59] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:59] Config Error: Bad data size 0 required 344
[2023.01.28 16:52:59] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:52:59] Config Error: Bad data size 0 required 344
[2023.01.28 16:53:00] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:53:00] Config Error: Bad data size 0 required 344
[2023.01.28 16:53:00] Config Error: Failed to parse fire response JSON, error: illegal value
[2023.01.28 16:53:00] Config Error: Bad data size 0 required 344
[2023.01.28 16:53:03] Message Info: bad message notification received (error_code 33) for msg_id = 7193693049421969416, seq_no = 146
[2023.01.28 16:53:03] Message Info: bad message notification received, msgId 7193693049421969416, error_code 33
[2023.01.28 16:53:03] Message Info: bad message notification received (error_code 33) for msg_id = 7193693050111890428, seq_no = 282
[2023.01.28 16:53:03] Message Info: bad message notification received, msgId 7193693050111890428, error_code 33
[2023.01.28 16:53:04] MTP Info: -404 error received in 160005 with temporary key, assuming it was destroyed.
[2023.01.28 16:53:07] Message Info: bad message notification received (error_code 33) for msg_id = 7193693061976412940, seq_no = 54
[2023.01.28 16:53:07] Message Info: bad message notification received, msgId 7193693061976412940, error_code 33
```

**Comment by ortwinbachmann30** [comment]
I forgot to upload video.


https://user-images.githubusercontent.com/74976078/215269378-227bc9e3-845c-4069-9002-de70d783937f.mp4

**Comment by ortwinbachmann30** [comment]
Fixed Successfully in version 4.5.9 beta x64.
